### DISCUSSION TITLE
icloud emails not receiving sign in email link from Firebase

### DISCUSSION THREAD

**Discussion by matteoSANE** [firebase/flutterfire#11819] [Q&A]
Hi there! 

I've implemented email link authentication using Firebase Auth for Flutter on my mobile application, it's working fine with all emails but starting from today all users with icloud.com email couldn't receive their email link. I don't really know why. 

Could you help me about this? 
Thank you in advance.

Below my code ::

```
var acs = ActionCodeSettings(
        url: 'https://myapp.page.link/email_link/',
        handleCodeInApp: true,
        iOSBundleId: 'com.myApp.app',
        androidPackageName: 'com.myapp.app',
        // installIfNotAvailable
        androidInstallApp: true,
        // minimumVersion
        androidMinimumVersion: '12');

    try{
      await _auth.sendSignInLinkToEmail(
          email: email,
          actionCodeSettings: acs
      );

      UserBox().emailToAuth = email;

      return true;
    } catch(err){
      printLog('Error sending email verification $err');
      rethrow;
    }
```

**Comment by obrhoff** [comment]
Did you ever found a solution? Currently I'm struggling with the same issue.

**Comment by karims10** [comment]
Same here !

**Reply by obrhoff** [reply]
My only workaround was to go through Mailchimp, by using a custom SMTP in Firebase. I hope that helps.

**Reply by karims10** [reply]
Thanks for your response. Indeed I can confirm that it works well with a custom smtp. Thanks !

**Comment by cervonwong** [comment]
Still same issue here

**Comment by t0mstah** [comment]
Same issue here
